insights/ui/base.py

```diff
@@ -180,6 +180,25 @@ def wait_until_element_invisible(self, locator, timeout=12, poll_frequency=0.5):
             )
             return None
 
+    def wait_until_element_to_be_clickable(self, locator, timeout=12, poll_frequency=0.5):
+        try:
+            self.browser.implicitly_wait(3)
+            element = WebDriverWait(
+                self.browser, timeout, poll_frequency
+            ).until(
+                expected_conditions.element_to_be_clickable(locator),
+                message=u'Element is not clickable %s' % (locator[1])
+            )
+            return element
+        except TimeoutException as err:
+                self.logger.debug(
+                    u"%s: Waiting for text '%s' to be present under %s",
```

Comment:
Change logger message from Waiting for text to Waiting for element to be clickable.